tests/apps_test.go

```diff
@@ -108,6 +108,9 @@ func appsLogsTest(t *testing.T, params *utils.DeisTestConfig) {
 
 func appsOpenTest(t *testing.T, params *utils.DeisTestConfig) {
 	utils.CurlApp(t, *params)
+	cfg := utils.GetGlobalConfig()
+	cfg.AppName("ghostapp")
+	CurlWithFail(t, fmt.Sprintf("http://%s.%s", cfg.AppName, cfg.Domain), true, "404 Not Found")
```

Comment:
change to `CurlWithFail(t, fmt.Sprintf("http://%s.%s", "this-app-does-not-exist", cfg.Domain), true, "404 Not Found")`